tests/test_log_out.py

```diff
@@ -0,0 +1,16 @@
+from .pages.formLogin import LogIn
+from .pages.logout import Logout
+
+
+def test_log_in_success(driver):
```

Comment:
test_log_in_success - strange name, in this test you check logout, not test_log_in_success

Reply:
ok, is done